app/src/main/java/com/github/libretube/api/PlaylistsHelper.kt

```diff
@@ -133,7 +135,41 @@ object PlaylistsHelper {
         }
     }
 
-    suspend fun importPlaylists(playlists: List<ImportPlaylist>) = withContext(Dispatchers.IO) {
+    suspend fun importFreeTubePlaylists(playlists: List<FreeTubeImportPlaylist>) = withContext(Dispatchers.IO) {
```

Comment:
That's just a lot of duplicated code. Instead, please keep the old `importPlaylists` function and call it no matter if it's a Piped or a FreeTube playlist that's being imported.
That means we should "convert" the `List<FreeTubeImportPlaylist` to a `List<PipedImportPlaylist>` using 
```kotlin
freeTubePlaylists.map { PipedImportPlaylist(it.title, ...) }
```
(the above is just a simple example).